src/launcher/index.ts

```diff
@@ -123,6 +130,12 @@ async function main() {
         if (flowrWindow.store.get('clearAppDataOnStart')) {
           await clearBrowsingData()
         }
+
+        // If the browserWindow is not properly configured to use a player configured zPosition, destroy and remake it
+        if (flowrWindow.transparent && userPreferencesMerged.player.position !== PlayerPosition.BACKGROUND) {
```

Comment:
Shouldn't this also cover the opposite case ?
```suggestion
        if ((flowrWindow.transparent && userPreferencesMerged.player.position !== PlayerPosition.BACKGROUND)
            || !flowrWindow.transparent && userPreferencesMerged.player.position !== PlayerPosition.FOREGROUND
        ) {
```